recipy/log.py

```diff
@@ -96,17 +96,27 @@ def log_init():
     sys.excepthook = log_exception
 
 
-def log_values(custom_values_dict):
-    """ Log a dictionary of custom value-key pairs into the database """
-    # validation and debugging
-    assert isinstance(custom_values_dict, dict), \
-        "custom_values_dict must be a dict. type(custom_values_dict) = %s" % custom_values_dict
+def log_values(custom_values=None, **kwargs):
+    """ Log a dictionary of custom value-key pairs into the database
+    e.g,
+    >>> log_values(a=1, b=2)
+    >>> log_values({'c': 3, 'd': 4})
+    >>> log_values({'e': 5, 'f': 6}, g=7, h=8)
+    """
+
+    # create dictionary of custom values from arguments
+    custom_values = {} if custom_values is None else custom_values
+    assert isinstance(custom_values, dict), \
+        "custom_values must be a dict. type(custom_values) = %s" % custom_values
```

Comment:
Shouldn't that be `"custom_values must be a dict. type(custom_values) = %s" % type(custom_values)`?